src/index.tsx

```diff
@@ -19,6 +20,12 @@ setInterval(() => {
   store.dispatch(fetchPastelPrice())
 }, oneHour)
 
+try {
+  PastelDB.getDatabaseInstance()
+} catch (error) {
+  throw new Error('"pastelDB create:" error')
```

Comment:
If you re-throw the error here, on reason to use try/catch. The reason for it is to allow users to use the wallet even if there are problems creating DB
```suggestion
  // TODO log errors to a central logger so we can address them later.
  console.error(`PastelDB.getDatabaseInstance error: ${error.message}`)
```

Reply:
Could add the a `// TODO` as I asked please? If you check our codebase, this comment precedes all `console` statements

Reply:
okay, added.